File: Aigilas/Aigilas/Strategies/NullStrategy.cs
using Aigilas.Creatures;
using Aigilas.Skills;
using SPX.Core;

namespace Aigilas.Strategies
{
    public class NullStrategy : IStrategy
    {

        public NullStrategy(ICreature parent, params int[] targetTypes)
            : base(parent,Strategy.Null)
        {
            foreach (var targetType in targetTypes)
            {
                _targets.AddTargetTypes(targetType);
            }
        }

        public override void Act()
        {
            
        }
    }
}

File: proj/src/aigilas/strategies/impl/NullStrategy.java
package aigilas.strategies.impl;

import aigilas.creatures.BaseCreature;
import aigilas.strategies.BaseStrategy;
import spx.bridge.ActorType;

public class NullStrategy extends BaseStrategy {

    public NullStrategy(BaseCreature parent, ActorType... targetTypes)

    {
        super(parent, null);

        for (ActorType targetType : targetTypes) {
            _targets.addTargetTypes(targetType);
        }
    }

    @Override
    public void act() {

    }
}
